runelite-api/src/main/java/net/runelite/api/AnimationID.java

```diff
@@ -59,6 +59,31 @@ public final class AnimationID
 	public static final int WOODCUTTING_2H_CRYSTAL = 10072;
 	public static final int WOODCUTTING_2H_CRYSTAL_INACTIVE = 10073;
 	public static final int WOODCUTTING_2H_3A = 10074;
+	/*Ent animation shared with Canoe cutting*/
+	public static final int WOODCUTTING_ENT_BRONZE = 3291;
+	public static final int WOODCUTTING_ENT_IRON = 3290;
+	public static final int WOODCUTTING_ENT_STEEL = 3289;
+	public static final int WOODCUTTING_ENT_BLACK = 3288;
+	public static final int WOODCUTTING_ENT_MITHRIL = 3287;
+	public static final int WOODCUTTING_ENT_ADAMANT = 3286;
+	public static final int WOODCUTTING_ENT_RUNE = 3285;
+	public static final int WOODCUTTING_ENT_DRAGON = 3292;
+	public static final int WOODCUTTING_ENT_CRYSTAL = 8327;
+	public static final int WOODCUTTING_ENT_CRYSTAL_INACTIVE = 8323;
+	public static final int WOODCUTTING_ENT_GILDED = 8305;
+	public static final int WOODCUTTING_ENT_3A = 7266;
+	public static final int WOODCUTTING_ENT_INFERNAL = 8780;
+	public static final int WOODCUTTING_ENT_2H_BRONZE = 10517;
+	public static final int WOODCUTTING_ENT_2H_IRON = 10518;
+	public static final int WOODCUTTING_ENT_2H_STEEL = 10519;
+	public static final int WOODCUTTING_ENT_2H_BLACK = 10520;
+	public static final int WOODCUTTING_ENT_2H_MITHRIL = 10521;
+	public static final int WOODCUTTING_ENT_2H_ADAMANT = 10522;
+	public static final int WOODCUTTING_ENT_2H_RUNE = 10523;
+	public static final int WOODCUTTING_ENT_2H_DRAGON = 10524;
+	public static final int WOODCUTTING_ENT_2H_CRYSTAL = 10526;
```

Comment:
This seems to be backwards; 10525 is the active crystal axe, 10526 is the inactive axe.